content/blog/2016/2016-09-28-jenkins-world-2016-wrap-up-hub.adoc

```diff
@@ -0,0 +1,149 @@
+  plugins and link:http://www.meetup.com/St-Petersburg-Jenkins-Meetup/[St.
+  Petersburg Jenkins Area Meetup] organizer.
+* *link:https://github.com/bitwiseman[Liam Newman]* -
+  co-organizer link:http://www.meetup.com/Los-Angeles-Jenkins-Area-Meetup/[Seattle Jenkins Area Meetup],
+  contributor, and JobDSL and Pipeline user.
+* *link:https://github.com/orrc[Christopher Orr]* - Maintainer of multiple
+  Android-related plugins, including the
+  link:https://wiki.jenkins-ci.org/display/JENKINS/Android+Emulator+Plugin[Android Emulator plugin]
+  and contributor to numerous projects behind the scenes of Jenkins.
+* *link:https://github.com/cvega[Casey Vega]* - Organizer for the
+  link:http://www.meetup.com/Los-Angeles-Jenkins-Area-Meetup/[Los Angeles
+  Jenkins Area Meetup] and release engineer at Verizon Digital Media.
+* *link:https://github.com/markewaite[Mark Waite]* - Maintainer of the
+  link:http://wiki.jenkins-ci.org/display/JENKINS/Git%20Plugin[Git plugin] and
+  contributor to a number of other Git-related plugins.
+* *link:https://github.com/dty[Dean Yu]* - Long-time contributor, board member
+  and release engineer at Shutterfly.
+* *link:https://github.com/kzantow[Keith Zantow]* - contributor on the
+  link:https://wiki.jenkins-ci.org/display/JENKINS/Blue+Ocean+Plugin[Blue Ocean plugin]
+  and overall UX effort.
+
+By choice, I actually spent the majority of my time at the booth. It was
+fantastic to hear how people were using
+Jenkins and wanting to use Jenkins to do more. I answered dozens of questions
+on both days of the conference, often learning new things in the process of answering them.
+And for questions that were beyond any one person's knowledge, there was such a
+breadth of expertise, very few questions were beyond our combined abilities.
+
+While the Open Source Hub was generally very well attended, the lunch-time demos drew
+really big crowds.
+
+NOTE: a bit more text here about the demos in general.
+
+NOTE: still working on this list
+
+* link:https://github.com/kzantow[Keith Zantow] gave a live demo of
+  *link:/projects/blueocean[Blue Ocean] in Action*.
+* link:https://github.com/bitwiseman[Liam Newman] gave a live demo of
+  *Notifications with Jenkins Pipeline* based on his previous
+  link:/blog/2016/07/18/pipline-notifications/[blog post].
+* link:https://github.com/orrc[Christopher Orr] presented a lightning version of his
+  *Pipelines for building and deploying Android apps* talk
+  (link:link:https://www.cloudbees.com/sites/default/files/2016-jenkins-world-continuous_build_delivery_pip.pdf[Slides]).
+* link:https://github.com/oleg-nenashev[Oleg Nenashev] showed a different way to
+  manage security with the *Ownership plugin for Jenkins*
+  (link:https://speakerdeck.com/onenashev/jw2016-ownership-plugin-demo[Slides])
+
+| 12:45 - 13:00
+| *Docker and Pipeline*
+| _Learn how to use Docker inside of Pipeline for clean, repeatable testing environments_
```

Comment:
My slides should be available as PDFs, if they're not hosted on the CloudBees site, you should have access to them and can drop them in `content/files` somewhere